docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -0,0 +1,437 @@
+- navigate to a folder page where you can assign folder and dashboard permissions
+
+### Dashboard folder page
+
+You can complete the following tasks on the **Dashboard Folder** page:
+
+- Move or delete dashboards in a folder
+- Rename a folder (available under the **Settings** tab)
+- Assign permissions to folders (which are inherited by the dashboards in the folder)
+
+To navigate to the dashboard folder page, click the cog appears when you hover over a folder in the dashboard search result list or the **Manage dashboards and folders** page.
+
+### Dashboard permissions
+
+You can assign permissions to a folder. Any permissions you assign are inherited by the dashboards in the folder. An Access Control List (ACL) is used where **Organization Role**, **Team** and a **User** can be assigned permissions.
+
+For more information about dashboard permissions, refer to [Dashboard permissions]({{< relref "../../administration/roles-and-permissions/#dashboard-permissions" >}}).
+
+## Export and import dashboards
+
+You can use the Grafana UI or the [HTTP API]({{< relref "../../developers/http_api/dashboard/#create-update-dashboard" >}}) to export and import dashboards.
+
+### Export a dashboard
+
+The dashboard export action creates a Grafana JSON file that contains everything you need, including layout, variables, styles, data sources, queries, and so on, so that you can later import the dashboard.
+
+1. Open the dashboard you want to export.
+2. Click the **Share** icon.
+3. Click **Export**.
+4. Click **Save to file**.
+
+Grafana downloads a JSON file to your local machine.
+
+{{< figure src="/static/img/docs/export/export-modal.png" max-width="800px" >}}
+
+#### Make a dashboard portable
+
+If you want to export a dashboard for others to use, you can add template variables for things like a metric prefix (use a constant variable) and server name.
+
+A template variable of the type `Constant` will automatically be hidden in the dashboard, and will also be added as a required input when the dashboard is imported.
+
+### Import a dashboard
+
+1. Click **Dashboards > Import** in the side menu.
+
+   {{< figure src="/static/img/docs/v70/import_step1.png" max-width="700px" >}}
+
+1. Perform one of the following steps:
+
+   - Upload a dashboard JSON file
+   - Paste a [Grafana.com](https://grafana.com) dashboard URL
+   - Paste dashboard JSON text directly into the text area
+
+{{< figure src="/static/img/docs/v70/import_step2_grafana.com.png"  max-width="700px" >}}
+
+The import process enables you to change the name of the dashboard, pick the data source you want the dashboard to use, and specify any metric prefixes (if the dashboard uses any).
+
+### Discover dashboards on Grafana.com
+
+Find dashboards for common server applications at [Grafana.com/dashboards](https://grafana.com/dashboards).
+
+{{< figure src="/static/img/docs/v50/gcom_dashboard_list.png" max-width="700px" >}}
+
+### Import and sharing with Grafana 2.x or 3.0
```

Comment:
@torkelo , this section seems rather old. Should we remove it?

Reply:
yes, you can remove this section

Reply:
Done.